apps/web/components/v2/settings/teams/DisableTeamImpersonation.tsx

```diff
@@ -0,0 +1,63 @@
+import { classNames } from "@calcom/lib";
```

Comment:
Had a call with @zomars and he mentioned we should move new components out of `apps/web/components` and into `packages/features`. Should @CarinaWolli do the same here?

Reply:
Agreed. Will try to do it myself.